osf/models/node.py

```diff
@@ -1450,6 +1450,7 @@ def register_node(self, schema, auth, draft_registration, parent=None, child_ids
             alternative_resource = None
 
         registered.copy_editable_fields(resource, auth=auth, alternative_resource=alternative_resource)
+        registered.copy_contributors_from(self)
```

Comment:
`copy_editable_fields` on the preceding line calls `copy_contributors_from`. What behavior do we want here?

If we want contributors copied from the the node, regardless of whether or not the registered node is a root or component, the `copy_editable_fields` call should include a `contributors=False` argument. 

Otherwise, if the registered node is the root, contributors will be copied from the draft_registration and then they will be copied from the node. If the registered node is not the root, `copy_contributors_from` will effectively run twice.

Reply:
I believe the case for existing draft_registrations with contributors (which were branched_from a node) is handled here if `contributors=False` is added as an argument for `copy_editable_fields` on L1452

Reply:
I think this additional copy contributors from was a leftover artifact from bug finding.